CONTRIBUTING.md

```diff
@@ -424,72 +440,84 @@ KUBERNETES_VERSION==v1.13.0 BACKEND=postgres ENV=kubernetes ./scripts/ci/local_c
 
 * PYTHON_VERSION might be one of 3.5/3.6
 * BACKEND might be one of postgres/sqlite/mysql
-* ENV might be one of docker/kubernetes
+* ENV might be one of docker/kubernetes/bare
 * KUBERNETES_VERSION - required for Kubernetes tests - currently KUBERNETES_VERSION=v1.13.0.
 
-The kubernetes env might not work locally as easily as other tests because it requires your host
-to be setup properly (specifically it installs minikube cluster locally on your host and depending 
+The following environments are possible:
+ * The "docker" environment (default) - it starts all dependencies required by full suite of tests
+   including postgres/mysql and other components. It's quite resource heavy so do not forget to stop
+   the environment using [Stop environment](#stopping-the-environment) when you finish working wit it. It
+   can also slow down your local machine.
+ * The "kubernetes" environment is used to run tests with kubernetes cluster.
+ * The "bare" environment is used to run a bare docker instance of Airflow without any external dependencies
```

Comment:
* Basic: This environment runs airflow in docker without any external dependencies. It will only work for non-dependent tests.